scripts/common.js

```diff
@@ -94,6 +94,19 @@ export function setThresholdsForScenarios (options) {
   }
 }
 
+export function getCountryCoordinates (countryCode, coordinatesEnv, useProto = false) {
+  const data = useProto ? protoCountries : countries
+
+  if (coordinatesEnv) {
+    const coords = coordinatesEnv.split(',')
+    if (coords.length === 2) {
+      const [lat, lng] = coords.map(Number)
+      return { lat, lng }
```

Comment:
hmm. I'm not sure here. Is it the same as {lat: lat, lng: lng} ?

Reply:
![image](https://github.com/traveltime-dev/traveltime-benchmarks/assets/140691866/d239b244-895f-41da-a51f-ad847dbcee88)
logged request body with -e COORDINATES=-8,13

Reply:
![image](https://github.com/traveltime-dev/traveltime-benchmarks/assets/140691866/e1f1ca6f-e38f-4d16-8044-8c9afab4f031)
and this is without the new env var

Reply:
fair enough, just making sure